example-doc-win2019.md

```diff
@@ -0,0 +1,261 @@
+# Windows Server 2019
+
+## Installed Software
+### Languages and Runtimes
+- Java 1.7: build 1.7.0_262-b10
+- Java 1.8: build 1.8.0_252-b14 (default)
+- Java 11: build 11.0.7+10-LTS
+- Rust 1.43.0
+- Python 3.7.7
+- Ruby 2.5.8p224
+- Go 1.14.2
+- PHP 7.4.5
+- julia 1.4.1
+- Perl 5.30.2
+- TypeScript 3.8.3
```

Comment:
I wonder if we really need TypeScript in docs at all? TypeScript is not the tool. It is just node library